src/main/webapp/js/common.js

```diff
@@ -137,25 +137,14 @@ var TEAMNAME_MAX_LENGTH = 60;
 var NAME_MAX_LENGTH = 40;
 var INSTITUTION_MAX_LENGTH = 64;
 
-$(document).on('ajaxComplete ready', function() {
-    /**
-     * Initializing then disabling is better than simply
-     * not initializing for mobile due to some tooltips-specific
-     * code that throws errors.
-    */
-    var $tooltips = $('[data-toggle="tooltip"]');
-    $tooltips.tooltip({html: true, container: 'body'});
-    if (isTouchDevice()) {
-        $tooltips.tooltip('disable');
-    }
-});
+var util = {
 
 /**
  * Checks if the current device is touch based device
  * Reference: https://github.com/Modernizr/Modernizr/blob/master/feature-detects/touchevents.js
  */
-function isTouchDevice() {
```

Comment:
Could you change the indentation to make it follow our Javascript Style Guide? https://docs.google.com/document/d/1gZ6WG6HBTJYHAtVkz9kzi_SUuzfXqzO-SvFnLuag2xM/pub?embedded=true